src/components/Embeds/AirGraph.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import QualityStatsGraph from '../City/QualityStatsGraph';
+
+const DATE_FMT_OPTIONS = {
+  timeZone: 'UTC',
+  weekday: 'short',
+  day: 'numeric',
+  month: 'short'
+};
+
+class AirGraph extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stats: []
+    };
+  }
+
+  componentDidMount() {
+    const { location } = this.props;
+    const params = new URLSearchParams(location.search);
+    const city = params.get('city');
+
+    const formatWeeklyP2Stats = data => {
```

Comment:
Lets find a way to centralize these calculations... May be move them into the `QualityStatsGraph` component and just pass data from this component and `City` page component?.

Reply:
😢  okay